api/graphql/tests/test_reservations/test_reservation_create.py

```diff
@@ -1466,57 +1472,6 @@ def test_create_price_calculation_with_time_based_price_reservation_unit(
         )  # 1h reservation = 4 x 15 min = 4 x 3 €
         assert_that(reservation.tax_percentage_value).is_equal_to(tax_percentage.value)
 
-    def test_create_price_calculation_rounding(self, mock_periods, mock_opening_hours):
```

Comment:
Removed this test because it is pointless. With the new validation we can't have a reservation that is not a multiple of interval.